bootstrap.sh

```diff
@@ -9,21 +9,30 @@ if [ "$1" = "--skip_root_installs" ]; then
   SKIP_ROOT_INSTALLS=True
 fi
 
+# Exits if $1 is not 0 and prints $2 as error message.
```

Comment:
We had a pattern before where there's a "fail" function that prints the message and exits. You would use it like this:

``` sh
do something || fail "exit message"
```

I think this is more idiomatic BASH (if there is such a thing).